staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/helpers.go

```diff
@@ -43,14 +43,17 @@ func NestedFieldCopy(obj map[string]interface{}, fields ...string) (interface{},
 
 func nestedFieldNoCopy(obj map[string]interface{}, fields ...string) (interface{}, bool, error) {
 	var val interface{} = obj
+	var evaluatedFields []string
```

Comment:
Most of the time size of this slice will match `len(fields)` so could be pre-allocated.

Reply:
Done.